bcgov/HOUS-permit-portal
# Feature/bphh 761/my submitted

## Description

Adds PA index filter sections for statuses. All properly integrated with the query params in the URL!
Also fix the logo blurryness

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-761


## Steps to QA

Reset DB with new seeds
Log in as submitter
Go my PAs
try out the tabs at the top
Complete some permit applications and see them move from the draft tab to the submitted tab
